src/core/glyphlist.js

```diff
@@ -13,11 +13,10 @@
  * limitations under the License.
  */
 /* no-babel-preset */
-/* eslint-disable no-var */
```

Comment:
*Including this here for future reference, to clarify that we've considered the implications of these changes:*

Here, and in `src/core/unicode.js`, we include these files as-is in the "legacy"-builds because of issues with Babel; please note https://github.com/mozilla/pdf.js/blob/052db45e7a26fead95b5233e3425008784daa8cf/gulpfile.js#L198-L200

Hence this PR will result in `let`/`const` being included even in the "legacy"-builds, however that should be fine given the following compatibility information:
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#browser_compatibility
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#browser_compatibility

Note also that IE 11 support is now explicitly removed, and this CSS-compatibility configuration for the `GENERIC`-builds: https://github.com/mozilla/pdf.js/blob/052db45e7a26fead95b5233e3425008784daa8cf/gulpfile.js#L78-L90